alphamountain/info.json

```diff
@@ -0,0 +1,296 @@
+{
+  "name": "alphamountain",
+  "label": "alphaMountain",
+  "version": "1.0.0",
+  "category": "Threat Detection",
+  "description": "alphaMountain Threat Response integrates investigations informed by reputation scores of target hosts, domains, and IP addresses. It fetches indicators with risk scores and relevant content categorization.",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "publisher": "Fortinet",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
```

Comment:
Doc link is missing.

Reply:
Will add the Doc link once the InfoDev team provides it.